cocotb/handle.py

```diff
@@ -220,7 +219,7 @@ def _discover_all(self):
             name = thing.get_name_string()
             try:
                 hdl = SimHandle(thing, self._child_path(name))
```

Comment:
This would be clearer as:
```suggestion
            path = self._child_path(name)
            try:
                hdl = SimHandle(thing, path)
```
Otherwise there's a question of which bit is failing

Reply:
right, will change